assets/keymaps/default-macos.json

```diff
@@ -402,7 +402,7 @@
       "alt-cmd-]": "editor::UnfoldLines",
       "ctrl-space": "editor::ShowCompletions",
       "cmd-.": "editor::ToggleCodeActions",
-      "alt-cmd-r": "editor::RevealInFinder",
+      "alt-cmd-r": "editor::RevealInFileManager",
```

Comment:
Do we have to do this still? I think one point behind @mikayla-maki saying that we don't rename is that we also don't break the keybindings.

Reply:
Fixing